### PR TITLE
Update benchmark scripts

### PR DESCRIPTION
**Pull request by Jooyuza**
- Add a warmup for benchmark scripts, especially for np.vectorize().

- Add torontonian_cuda_deepquantum since the gpu acceleration is significant.

### CODE REVIEW

File: examples/benchmarks/benchmark_v420/torontonian/torontonian_cuda_deepquantum.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+import deepquantum as dq
+import torch
+from deepquantum.photonic.torontonian_ import torontonian
+from tqdm import tqdm
+
+# Print version
+print(dq.__version__)
+
+device = 'gpu'
```
Comment (sansiro77) [inline examples/benchmarks/benchmark_v420/torontonian/torontonian_cuda_deepquantum.py]: cuda

**Review by sansiro77 — APPROVED** [review]
LGTM
